core/src/main/scala/org/http4s/UriTemplate.scala

```diff
@@ -100,7 +100,7 @@ object UriTemplate {
 
   type Path = List[PathDef]
   type Query = List[QueryDef]
-  type Fragment = List[FragmentDef]
+  type FragmentDefs = List[FragmentDef]
```

Comment:
I'm not sure I like this type alias at all.  I certainly didn't like the old alias.